onboard-jmuniz/constants/interfaces/login-user-response-props.ts

```diff
@@ -0,0 +1,5 @@
+export interface loginUserResponseProps {
```

Comment:
```suggestion
export interface LoginUserResponseProps {
```

Por convenção utilizamos nomes de interface com letra maiúscula.